rest_api/controller/file_upload.py

```diff
@@ -49,7 +49,7 @@ def file_upload(
         meta["name"] = file.filename
 
         INDEXING_PIPELINE.run(
-            file_path=file_path,
+            file_paths=file_path,
```

Comment:
Is there a way for us to support batch file uploading in the REST API as well?

Reply:
I pushed the changes please review it.